napari/viewer.py

```diff
@@ -84,9 +84,9 @@ def screenshot(self, path=None, *, canvas_only=True):
             image = self.window.screenshot(path=path)
         return image
 
-    def show(self):
+    def show(self, run=False):
```

Comment:
See question about what does `run=False` mean with `show` on the main window

Reply:
commented above